conftest.py

```diff
@@ -22,6 +26,26 @@ def driver() -> WebDriver:
     LOGGER.info(f'Driver quit')
 
 
+@pytest.fixture(scope='session')
+def clear_initialize_db():
+    api_url = API_URL
+    base_api.api_post(f'{api_url}/cleanDB')
+    base_api.api_post(f'{api_url}/initializeDB')
+
+
+@pytest.fixture(scope='session')
+def user_scenario():
+    return (Dict(customer_id=12434, user_data=Dict(repeatedPassword='password',
```

Comment:
Constant of the static user should be stored in a constant (or, better to get it from API, for example
curl -X GET --header "Accept: application/json" "[http://localhost:8080/parabank/services/bank/login/${username}/${password}](http://localhost:8080/parabank/services/bank/login/$%7Busername%7D/$%7Bpassword%7D)")